specs/pvs/corla.pvs

```diff
@@ -863,6 +934,8 @@ BEGIN
 % @description The audited contest determines the number of ballots
 % that must be examined and verified during the RLA.
   number_of_ballots_to_audit: [audited_contest -> nat]
+
+% @todo kiniry Still need to model core RLA algorithm(s).
 END rlas
 
 %+END audits
```

Comment:
We don't know.  Again, this is mentioned as an adjective in C.R.S., but we do not have clarity on what it means.